sklearn/feature_extraction/text.py

```diff
@@ -1008,6 +1026,8 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     Normalization is "c" (cosine) when ``norm='l2'``, "n" (none)
     when ``norm=None``.
 
+    By default, the l2 norm is used.
```

Comment:
This should be at the explanation of the norm parameter, the first line should end with something like (default='l2')